File: Chicken/Scripts/app/model/Device.js
Ext.define('MyApp2.model.Device', {
    extend: 'Ext.data.Model',
    fields: [
        { name: 'deviceId', type: 'string', convert: null },
        { name: 'deviceNo', type: 'string' },
        { name: 'name', type: 'string' },
        { name: 'alias', type: 'string', convert: null },
        { name: 'gatewayId', type: 'string' },
        { name: 'parentId', type: 'string' },
        { name: 'status', type: 'string', convert: null },
        { name: 'device_type', type: 'string' },
        { name: 'vendor', type: 'string' },
        { name: 'brand', type: 'string', convert: null },
        { name: 'model', type: 'string' },
        { name: 'rom_version', type: 'string' },
        { name: 'network_type', type: 'string', convert: null },
        { name: 'mac', type: 'string' },
        { name: 'private_ip', type: 'string' },
        { name: 'public_ip', type: 'string', convert: null },
        { name: 'ssid', type: 'string' },
        { name: 'address', type: 'string' },
        { name: 'config', type: 'string', convert: null },
        { name: 'comment', type: 'string' },
        { name: 'offtime', type: 'date', dateFormat: 'Y-m-d' },
        { name: 'onlinetime', type: 'date', dateFormat: 'Y-m-d' },
        { name: 'createtime', type: 'date', dateFormat: 'Y-m-d' },
        { name: 'token', type: 'string' },
        { name: 'projectId', type: 'string' },
        { name: 'is_gateway', type: 'string', defaultValue: true }
    ]

});

File: Chicken/Infrastructure/device.cs
using Newtonsoft.Json;
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Data.Entity.Spatial;

namespace Chicken.Infrastructure
{
    [Table("device")]
    public partial class Device
    {
 
        public Device()
        {
            point = new HashSet<Point>();
            role_device = new HashSet<Role_Device>();
        }

        public string deviceId { get; set; }

        [Required]
        [StringLength(128)]
        public string projectId { get; set; }

        [Required]
        [StringLength(128)]
        public string deviceNo { get; set; }

        [Required]
        [StringLength(256)]
        public string name { get; set; }

        [StringLength(256)]
        public string alias { get; set; }

        [StringLength(128)]
        public string parentId { get; set; }

        [StringLength(128)]
        public string gatewayId { get; set; }

        public bool? is_gateway { get; set; }

        [StringLength(256)]
        public string token { get; set; }

        [StringLength(256)]
        public string status { get; set; }

        public DateTime? offtime { get; set; }

        public DateTime? onlinetime { get; set; }

        public DateTime? createtime { get; set; }

        [StringLength(256)]
        public string device_type { get; set; }

        [StringLength(256)]
        public string vendor { get; set; }

        [StringLength(256)]
        public string brand { get; set; }

        [StringLength(256)]
        public string model { get; set; }

        [StringLength(256)]
        public string rom_version { get; set; }

        [StringLength(256)]
        public string network_type { get; set; }

        [StringLength(256)]
        public string mac { get; set; }

        [StringLength(256)]
        public string private_ip { get; set; }

        [StringLength(256)]
        public string public_ip { get; set; }

        [StringLength(256)]
        public string ssid { get; set; }

        [StringLength(256)]
        public string bssid { get; set; }

        [StringLength(256)]
        public string wifi_password { get; set; }

        [StringLength(256)]
        public string address { get; set; }

        [StringLength(256)]
        public string config { get; set; }

        [StringLength(1024)]
        public string comment { get; set; }
        [JsonIgnore]
        public virtual Project project { get; set; }

        public virtual ICollection<Point> point { get; set; }
        [JsonIgnore]
        public virtual ICollection<Role_Device> role_device { get; set; }
    }
}
